recipe_scrapers/__init__.py

```diff
@@ -662,4 +685,4 @@ def scrape_html(
 
 
 __all__ = ["scrape_me", "scrape_html"]
-name = "recipe_scrapers"
+name = "recipe-scrapers"
```

Comment:
Was this naming adjustment required for some reason?  For safety/compatibility, I'd prefer not to modify it unless we need to.

Reply:
That was a mistake from when I refactored my pycharm at one point - I'll go ahead and fix it in the next commit. Apologies!